index.html

```diff
@@ -1238,10 +1238,33 @@
       <section>
         <h3>Atomize Information for Principle of Least Authority</h3>
 
-        <p class="issue">
-When issuing information, issue it in the smallest pieces possible.
-When requesting information, only ask for the data points that you need.
-Issue abstract claims alongside actual data when possible.
+        <p>
+Privacy issues arise when issuers bundle too much information into a
+claim set. For example, a drivers license containing a persons ID number,
+height, weight, birthday, and home address is an example of a claim set
+containing far too much information if one of the intended usages of the
```

Comment:
`far too much...` => `more information than is necessary to establish that the person is above a certain age.`